amethyst_gltf/src/systems.rs

```diff
@@ -0,0 +1,410 @@
+use assets::{AssetStorage, Handle, Loader};
+use core::transform::*;
+use renderer::{Material, MaterialDefaults, Mesh, Texture};
+use specs::{Entities, Entity, Fetch, FetchMut, Join, System, WriteStorage};
+use specs::common::Errors;
+
+use {GltfMaterial, GltfPrimitive, GltfSceneAsset};
+
+/// A GLTF scene loader, will transform `Handle<GltfSceneAsset>` into full entity hierarchies.
+///
+/// Will also do the asset storage processing for `GltfSceneAsset`.
+pub struct GltfSceneLoaderSystem;
```

Comment:
Please add a dummy field in case we want to add fields later

Reply:
What is the recommended type for a dummy field?

Reply:
`()` I guess